lxd/storage/drivers/driver_zfs.go

```diff
@@ -25,6 +25,8 @@ var zfsTrim bool
 var zfsRaw bool
 
 var zfsDefaultSettings = map[string]string{
+	"atime":      "off",
```

Comment:
I wonder if we should also apply these settings on pool `Mount()` so they are applied to existing pools on next LXD restart?

Reply:
If we do that, I suspect it will work against those wanting absolutely no atime/relatime (like me). Also, this would set the property on each ZFS filesystem rather than have it inherited from the zpool config.

Reply:
Makes sense thanks.